tests/phpunit/tests/comment/wpComment.php

```diff
@@ -62,4 +62,45 @@ public function test_get_instance_should_succeed_for_float_that_is_equal_to_post
 
 		$this->assertEquals( 1, $found->comment_ID );
 	}
+
+
+	/**
+	 * @dataProvider data_test_wp_comment_null_byte
+	 * @ticket 52738
```

Comment:
```suggestion
	 * @ticket 52738
	 *
	 * @covers WP_Comment::__construct
	 *
	 * @dataProvider data_test_wp_comment_null_byte
	 *
	 * @param stdClass $comment_data The comment's data.
	 * @param string   $expected     The expected comment content.
```
- See [this ticket](https://core.trac.wordpress.org/ticket/56070) for the order of annotations in the test suite.
- A `@covers` annotation should be used.
- Test method parameters should be documented.

🔢 Applies elsewhere in the PR.